test/python/test_operation.py

```diff
@@ -323,5 +323,30 @@ def test_LeakyRelu(self):
         self.check_shape(dx.shape(), (3, 2))
 
 
+    def test_Elu_cpu(self):
+        #f(x) = alpha * (exp(x) - 1.) for x < 0, f(x) = x for x >= 0
+        x = np.array([-0.9, -0.3, -0.1, 0.1, 0.5, 0.9]).reshape(3, 2).astype(np.float32)
+        y = np.clip(x, 0, np.inf) + (np.exp(np.clip(x, -np.inf, 0)) - 1) * 1.0
+        x = tensor.from_numpy(x)
+        x.to_device(cpu_dev)
+
+        result = autograd.elu(x)
+        dx = result.creator.backward(x.data)
+
+        np.testing.assert_array_almost_equal(tensor.to_numpy(result), y, decimal=5)
```

Comment:
only forward is tested..
how about the backward?

Reply:
yes, I added backward test